functions/Get-DbaSpn.ps1

```diff
@@ -71,76 +68,81 @@ Returns a custom object with SearchTerm (ServerName) and the SPNs that were foun
 			}
 		}
 		
-		ForEach ($account in $AccountName)
+		if ($ShowAccountNameAll)
```

Comment:
Please use regions to split different sections of your code.
When using PSS:
Mark all of one logical section.
Press "CTRL" + "R"
Give it an acceptable name.
Makes it easier to read and maintain